### ISSUE TITLE
Pulled down masks are invisible

### ISSUE THREAD

**Issue by Errant-4** [space-wizards/space-station-14#30701]
## Description
Worn masks can be toggled, removing them from your mouth. We actually have sprites for this pulled down state but it's not displayed, the masks become invisible instead. 

Tested with Breath Mask on Human, presumably happens on every mask

-Put on your breath mask
-Click the mask toggle on Action Bar
-No visible mask.

**Comment by bruhmogus** [comment]
true, also when a fix is made we should probably make sure cham masks dont break
